charts/redhat/redhat/redhat-dotnet-imagestreams/0.0.1/src/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+description: |-
+  This content is experimental, do not use it in production. Build and run .NET applications on UBI.
```

Comment:
> This content is experimental, do not use it in production.

I've copied this description from the nodejs chart.

Similar, the _(experimental)_ in the `charts.openshift.io/name`.